pipeline/transformer/transformer_sierra/src/test/scala/weco/pipeline/transformer/sierra/transformers/SierraNotesTest.scala

```diff
@@ -423,6 +424,50 @@ class SierraNotesTest
     }
   }
 
+  describe("lettering for visual material") {
+    it("uses both 246 .6 ǂa and 514 for visual material") {
```

Comment:
Should this be something like "Does not create a lettering note for visual material"?

Reply:
I've tidied these up